internal/pkg/core/statsmanager.go

```diff
@@ -325,32 +406,106 @@ func (StatMgr *OpenOltStatisticsMgr) collectPONMetrics(pID uint32) map[string]fl
 
 	if metricName != nil && len(metricName) > 0 {
```

Comment:
why we need two methods collectNNIMetrics and collectPONMetrics ?It can be achieved with one method nly.